backend.eslint.config.mjs

```diff
@@ -0,0 +1,91 @@
+import antfu from "@antfu/eslint-config";
+
+export default antfu({
+  ignores: [
+    ".next/",
+    "**/.next/**/",
+    "build/",
+    "**/build/**/",
+    "dist/",
+    "**/dist/**/",
+    "out/",
+    "**/out/**/",
+    "public/",
+    "**/public/**/",
+    "**/next.config.js",
+    "**/vite.config.ts",
+    "**/vite-env.d.ts",
+  ],
+
+  // Disables stylistic rules to avoid conflicts with Prettier
+  stylistic: false,
+
+  // Enables type aware rules
+  typescript: {
+    tsconfigPath: "tsconfig.json",
+    overrides: {
+      // Avoid bugs
+      "ts/no-shadow": ["error", { ignoreTypeValueShadow: true }],
+      "ts/no-unsafe-unary-minus": "error",
+      "ts/no-unused-expressions": "error",
+      "ts/switch-exhaustiveness-check": "error",
```

Comment:
Do we need to specify this one as `error` even though we disable it later in the file? (This goes for all 3 of the new `eslint.config.mjs` files)